src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java

```diff
@@ -371,6 +415,21 @@ public void traverse() {
         engine.traverse();
      }
 
+    @Override
+    public Object onTraversalSuccess() {
+        if (costObservabilityTableName != null) {
+            CostObservability costObservability = new CostObservability(projectID, datasetID, costObservabilityTableName);
+            // TODO - is this needed?
+//            costObservability.writeCostObservability(callSetIdentifier, wdlStep, wdlCall, shardIdentifier,
+//                    new Date(), new Date(), "BigQuery Query Scanned",
+//                    sampleList.getBigQueryQueryByteScanned() + engine.getBigQueryQueryByteScanned());
```

Comment:
I dont think we still have a query cost for the extractCohort step (just the prepare step before it)
We used to, but now the querying is done as part of the prepare step and for the extract, we just grab everything from the prepare tables with the readAPI

Reply:
Thanks!